packages/alfa-act/src/question.ts

```diff
@@ -9,38 +9,49 @@ import * as json from "@siteimprove/alfa-json";
 /**
  * @public
  */
-export class Question<Q, S, A, T = A>
+export class Question<Q, S, C, A, T = A>
```

Comment:
Looks like in most case `C` and `S` are the same. Can't really default here due to parameter order, however 😞 Might be worth reordering parameters to avoid copying stuff around at call sites.